tools/install.sh

```diff
@@ -38,7 +38,8 @@
 set -e
 
 # Default settings
-ZSH=${ZSH:-~/.oh-my-zsh}
+ZDOTDIR=${ZDOTDIR:-~}
+ZSH=${ZSH:-${ZDOTDIR}/.oh-my-zsh}
```

Comment:
I'd rather keep this in a single line so we don't define `$ZDOTDIR` and we can later check if it was defined previous to the installer running.